catalyst/contrib/dl/callbacks/trace.py

```diff
@@ -0,0 +1,138 @@
+from typing import Union  # isort:skip
+from operator import lt, gt
+
+import warnings
+
+from pathlib import Path
+
+from catalyst.dl import Callback, CallbackOrder, State
+from catalyst.dl.utils import (
+    trace_model_from_state,
+    save_traced_model,
+)
+
+
+class TracerCallback(Callback):
+
+    def __init__(
+            self,
+            metric_key: str,
+            stage: str = None,
+            mode: str = "max",
```

Comment:
could we use the same logic for model selection as `EarlyStoppingCallback` one?
btw, is it possible to trace just last epoch model?

Reply:
I did the same logic as in `EarlyStoppingCallback`, but without `patience`.
Regarding last epoch only - am I understood correctly, that I just need to trace model at the end of the stage (or maybe loader)?

Reply:
I think, with `mode=last` it's better to trace model after each epoch.
In this way, you could automatically place it to you serving directory and update working microrservice. In theory :)

Reply:
Now I am a bit stuck :)
`mode="last"` - model will be traced only once on last epoch,
`mode="best"` - model will be traced every epoch metric improved,
is that correct? :)

Or do you want me to add logic to trace model only once but also for the best checkpoint?